07/reshape-a-matrix.checkpoint.md

```diff
@@ -109,14 +109,25 @@ def test_cannot_reshape():
 
     assert reshaped_matrix == [[1,2],[3,4]]
 
-def test_convert_2_2_to_1_4():
+def test_convert_4_2_to_2_4():
     matrix = [[1,2],[3,4],[5,6],[7,8]]
     r = 2
     c = 4
 
     reshaped_matrix = reshape_matrix(matrix, r, c)
 
     assert reshaped_matrix == [[1,2,3,4],[5,6,7,8]]
+
+def test_three_by_three_to_nine_by_one(self):
```

Comment:
Style nitpick: I personally like using the number words here over the numerals, but it might be good to be consistent with the other tests (which I recommend doing by updating the other tests to match this one).